File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/BillingPaymentMethodRequest.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * Payload for updating the default account payment method.
 */
class BillingPaymentMethodRequest
{
    /**
     * Payment method identifier (cc, paypal, or cc{index}).
     */
    #[DTA\Data(field: "payment_method", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $payment_method = null;

    /**
     * Whether automatic credit card payments are enabled.
     */
    #[DTA\Data(field: "cc_auto", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $cc_auto = null;

}

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/BillingPaymentMethodRequest.scala
package io.swagger.server.model


/**
 * = BillingPaymentMethodRequest =
 *
 * Payload for updating the default account payment method.
 *
 * @param payment_method Payment method identifier (cc, paypal, or cc{index}).
 * @param cc_auto Whether automatic credit card payments are enabled.
 */
case class BillingPaymentMethodRequest (
  payment_method: Option[String],
  cc_auto: Option[String]
)

